expression/builtin_time_vec.go

```diff
@@ -1358,11 +1359,40 @@ func (b *builtinTimestampDiffSig) vecEvalInt(input *chunk.Chunk, result *chunk.C
 }
 
 func (b *builtinUnixTimestampIntSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalInt evals a UNIX_TIMESTAMP(time).
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_unix-timestamp
 func (b *builtinUnixTimestampIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETDatetime, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil {
+		return err
+	}
```

Comment:
I've noticed that the previous implementation handles `ErrInvalidTimeFormat` here, 
https://github.com/pingcap/tidb/blob/e80bab6d46869a77d56bdaef29d950bcbd4fdc40/expression/builtin_time.go#L4320-L4323

Should we need to handle in vectorized implementation either?  @qw4990